app/components/new-reminder.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  store: Ember.inject.service(),
```

Comment:
Interesting choice sticking this in the component but I like that you're giving it a shot! This could also live in the controller where the store is readily available, and wouldn't require the `inject` loophole.